src/lib/commandBarConfigs/modelingCommandConfig.ts

```diff
@@ -505,6 +515,64 @@ export const modelingMachineCommandConfig: StateMachineCommandSetConfig<
       },
     },
   },
+  'Boolean Subtract': {
+    description: 'Subtract one solid from another.',
+    icon: 'booleanSubtract',
+    needsReview: true,
+    args: {
+      target: {
+        inputType: 'selection',
+        selectionTypes: ['path'],
+        selectionFilter: ['object'],
+        multiple: false,
+        required: true,
+        skip: true,
+        hidden: (context) => Boolean(context.argumentsToSubmit.nodeToEdit),
+      },
+      tool: {
+        clearSelectionFirst: true,
```

Comment:
Just a comment: I think this is pretty neat, thanks!

Reply:
Yeah, it good for now with what we have, in future it would be good to tell the engine what entities to make custom colours so that we can make these selection indicate that they're remembered, but not the current focus.